OPHD/States/CrimeRateUpdate.h

```diff
@@ -9,10 +9,13 @@ class Structure;
 class CrimeRateUpdate
 {
 public:
-	// Returns change in current morale caused by colony's mean crime rate
-	int update(const std::vector<TileList>& policeOverlays, PopulationPanel& populationPanel);
+	void update(const std::vector<TileList>& policeOverlays, PopulationPanel& populationPanel);
+
+	int getMoraleChange() const { return mMoraleChange; }
```

Comment:
I'd like @DanRStevens to weigh in here -- I know we've talked about verb-based function names but my general choice for most of the code for getters/setters is to drop the get/set prefix in favor of something like `int moraleChange() const 
 { return mMoraleChange; }`, basically treating it as a property versus a get/set type function.

I'm not going to be overly picky about it going forward but I would like to make a decision as to the preferred naming scheme here.

Something like the below `isProtectedByPolice` makes perfect sense versus `setPopulationPanel` which would probably be better served as `updatePopulationPanel`.

Thoughts?

Reply:
I have pretty mixed feelings on the naming convention. I do sort of like the shorter property naming convention, and was never very fond of "get" and "set", though C++ doesn't have property syntax support, so property uses are indistinguishable from normal function calls. That can be an issue because some noun phrases (properties) are easily confused with verb phrases (function calls), so it becomes very easy to mistake what a method does.

I think I would prefer verb phrases for functions and methods.

----

The `setPopulationPanel` method could probably have been named a bit better.